### PR TITLE
Added prettier

### PR DESCRIPTION
**Pull request by marioaks**
This commit adds prettier and styles all code on commit

### CODE REVIEW

**Review by alexwgraves — COMMENTED** [review]
Not sure this is really necessary when we're almost done and we're minimizing everything anyway...

File: src/css/map.css [hunk 1]
```diff
@@ -1,85 +1,70 @@
...
 
-.switch-container .switch.showing, .switch-container .switch:hover {
-  background-color: #1D1D1D;
+<<<<<<< HEAD
+.switch-container .switch.showing,
+.switch-container .switch:hover {
+  background-color: #1d1d1d;
 }
+=======
+>>>>>>> marioaks/table-tooltip
```
Comment (alexwgraves) [inline src/css/map.css]: Either way, you gotta fix this merge conflict before we can merge.

File: src/css/visualization.css [hunk 2]
```diff
@@ -94,22 +95,28 @@
...
 
 .viz-bar {
   position: relative;
   height: 0.75em;
+<<<<<<< HEAD
+  background-color: #00ed89;
+=======
   background-color: #00ED89;
+  /* border: 1px solid #000; */
+>>>>>>> marioaks/table-tooltip
```
Comment (alexwgraves) [inline src/css/visualization.css]: This one too

File: src/css/visualization.css [hunk 3]
```diff
@@ -221,15 +195,19 @@
...
   background-color: rgba(48, 48, 48, 1);
 }
 
 .ror-bar {
-  background-color: rgba(176, 140, 240, 1);
+  background-color: rgba(193, 78, 248, 1);
+<<<<<<< HEAD
+=======
+
+>>>>>>> marioaks/table-tooltip
```
Comment (alexwgraves) [inline src/css/visualization.css]: And this

**Review by alexwgraves — CHANGES_REQUESTED** [review]

**Comment by marioaks** [comment]
I've been working with VS code and a prettier plugin and I've been having to disable it every time I work on this project. Adding this in will honestly make my life way easier. Hope you're cool with it. Plus I don't really see a downside to it

**Review by alexwgraves — APPROVED** [review]
Alright, fine by me then!

**Pull request merged**